bin/cml/publish.js

```diff
@@ -43,7 +43,7 @@ exports.builder = (yargs) =>
         type: 'boolean',
         alias: 'gitlab-uploads',
```

Comment:
can we deprecate this (i.e. hide from the CLI `--help`)?

Reply:
`:relicanth:`

Reply:
Aliases can't be hidden, only removed

Reply:
well...

- create new hidden option + manual aliasing logic
- open/bump feature request on yargs?

Reply:
<a href="https://github.com/iterative/cml/pull/869#discussion_r786075314">

> — _should be aliased_ 
> — _Yes, but aliases can't be hidden. At least, I haven't found yet how to hide them._
> — _I'd rather let the alias show. Simplifies the code a lot. We could always leave out the alias in the online docs._

</a>

Two different opinions in three months... https://github.com/iterative/cml/pull/869#discussion_r786075314

Reply:
> _create new hidden option + manual aliasing logic_

Unless you've changed your mind, this approach hasn't improved since January. 😈

Reply:
> _open/bump feature request on yargs?_

That sounds great! 💯

Reply:
> Two different opinions in three months

not at all; the difference now is the flag no longer applies solely to GitLab, it also applies to Bitbucket.

Reply:
not at all; the _similarity_ is that we have some cache–invalidation–worthy flags and in both cases we're trying to find the best way of deprecating them. 🙃